app/generator/content/schedule/index.js

```diff
@@ -1,10 +1,12 @@
 const part1 = require('./part1')
-const summary = require('./summary')
+const part2 = require('./part2')
+const part3 = require('./part3')
 
 const createContent = (schedule) => {
```

Comment:
part1, part2 and part3 make sense for statements as they are lit. defined that way, i.e. `"Part 1: Payment summary"`

and I get the idea of splitting documents up into sep. parts, sorta hassa polymorphic feel but it's not as clear-cut where these splits are for a schedule document.

could we add a screenshot of a schedule with the 3 parts highlighted to help identify the boundaries of each?

Reply:
Screenshots now available here 
https://github.com/DEFRA/ffc-pay-statement-generator/tree/163639_Create_top-up_document_template_in_payment-generator_service/acceptance-test/2.1.0/schedule-output